Source/Parser/JsonLDParser.swift

```diff
@@ -128,18 +125,15 @@ class JsonLDParser {
 			} != nil
 	}
 	
-	static func parseAttributes(json: Dict, context: Context) -> (Attributes, Dict) {
+	static func parseAttributes(json: Dict, context: Context) -> (Attribute, Dict) {
 		let filteredJson = filterProperties(json: json, properties: ["@id", "@context", "@type"])
-		
+        
```

Comment:
While we don't have a defined SF, it may be easier to keep as it is for now. Can you revert this hunk?